### DISCUSSION TITLE
Add link to station in weekly update email

### DISCUSSION THREAD

**Discussion by golemk6** [mcguirepr89/BirdNET-Pi#1052] [Ideas]
This is a bit of an edge case, but just throwing it out there.

I am setting up a station for a marginally tach-savvy family member, and would like something to help keep it top of mind.  Weekly update emails would serve this purpose, but ideally there would be a link in these emails that, when clicked, would take them to the station web page.  How much of a challenge would this be?

**Comment by farski** [comment]
Adding the ability to send emails from the RPi probably wouldn't be too hard, but it would mean the installation has some credentials for an email server or service. For most people, it's probably not wise to give BirdNET their Gmail username and password, as these devices are generally connected to the Internet, and not really making any guarantees about security.

While it doesn't exactly solve your problem, and the emails wouldn't be able to include any specific stats or information from BirdNET, it would be trivial to use another service to send static recurring emails with a link to the BirdNET-Pi webpage. If you wanted to get clever, and the station is reporting to BirdWeather, it would not be hard to pull some data for the station from that to include in the email. It's a bit roundabout, but more you'll probably be waiting a while for local email support, and definitely more secure.

**Comment by golemk6** [comment]
I'm actually already using the build in Apprise functionality of BirdNETPi to send weekly summary emails through SendGrid, and those are coming through fine (see screenshot below).  What I was hoping for was an ability to add a custom header to those emails with a link to the BirdNETPi station.

Again, not a must have, but a would be nice to have.

![Clipboard01](https://github.com/mcguirepr89/BirdNET-Pi/assets/66495921/a78c5c16-e066-4fd9-a8a8-3982256ab47f)

**Comment by farski** [comment]
Ah got it, I knew Apprise was supported, but I'd never looked into it. Didn't realize it made sending emails so simple. One workaround you could try is patching [weekly_report.sh](https://github.com/mcguirepr89/BirdNET-Pi/blob/main/scripts/weekly_report.sh) on your relative's installation, if you have access. It wouldn't be hard to add another few lines to the body of the email by adding some text to the `NOTIFICATION` variable.
